src/lib/kdump/kdump_calibrator.rb

```diff
@@ -115,8 +143,11 @@ def total_memory
     #
     # @return [Hash] Memory limits
     def memory_limits
-      { min_low: min_low, max_low: max_low, default_low: default_low,
-        min_high: min_high, max_high: max_high, default_high: default_high }
+      {
+        min_low: min_low, max_low: max_low, default_low: default_low,
+        min_high: min_high, max_high: max_high, default_high: default_high,
+        min_fadump: min_fadump, max_fadump: max_fadump, default_fadump: default_fadump
```

Comment:
This reminds me that in Ruby 3.1 we can use shortcuts for Hash, https://rubyreferences.github.io/rubychanges/3.1.html#values-in-hash-literals-and-keyword-arguments-can-be-omitted

```suggestion
        min_low:, max_low:, default_low:,
        min_high:, max_high:, default_high:,
        min_fadump:, max_fadump:, default_fadump:
```

Reply:
yeah, sadly feature is for SLE15 SP5 so ruby 2.5